tests/release/e2e-test-default-with-deployment.go

```diff
@@ -168,7 +166,6 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deploy
 		if err != nil {
 			Expect(err.Error()).To(ContainSubstring("Reference does not exist"))
 		}
-		// TODO: Uncomment this lines once: https://issues.redhat.com/browse/RHTAPBUGS-409 is solved
```

Comment:
Why is this line being removed? It explains why the following is commented out.

Reply:
I'm fixing with some workaround